recipes-devtools/dmidecode/dmidecode_3.%.bbappend

```diff
@@ -0,0 +1,5 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/patches:"
+
+SRC_URI += " \
+    file://use-oe-vars.patch;patch=1 \
```

Comment:
When a file ends in .patch, Bitbake/OE treats it as a patch, so patch=1 isn't necessary.

Reply:
Yes, I took the original line, will amend.
(Btw, we should probably remove that everywhere when we can).